src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditDetails.tsx

```diff
@@ -1,19 +1,37 @@
 import { User } from 'shared';
-import { Controller, Control, FieldErrorsImpl } from 'react-hook-form';
-import { FormControl, FormLabel, Grid, MenuItem, TextField } from '@mui/material';
+import { FormControl, FormLabel, Grid } from '@mui/material';
 import PageBlock from '../../../layouts/PageBlock';
 import ReactHookTextField from '../../../components/ReactHookTextField';
 import AttachMoneyIcon from '@mui/icons-material/AttachMoney';
 import { fullNamePipe } from '../../../utils/pipes';
+import NERAutocomplete from '../../../components/NERAutocomplete';
 import { ProjectEditFormInput } from './ProjectEditContainer';
+import { Control, FieldErrorsImpl } from 'react-hook-form';
 
 interface ProjectEditDetailsProps {
   users: User[];
   control: Control<ProjectEditFormInput>;
   errors: FieldErrorsImpl<ProjectEditFormInput>;
+  projectManager: string | undefined;
+  projectLead: string | undefined;
+  setProjectManager: (projectManager?: string) => void;
+  setProjectLead: (projectLead?: string) => void;
 }
 
-const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({ users, control, errors }) => {
+const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({
+  users,
+  control,
+  errors,
+  projectManager,
+  projectLead,
+  setProjectLead,
+  setProjectManager
+}) => {
+  const userToAutocompleteOption = (user?: User): { label: string; id: string } => {
```

Comment:
move this method above the component since it's just acting as a helper

Reply:
we use this method so often we should extract it to a different file, export it and use the same one everywhere